src/VirtoCommerce.PricingModule.Core/Model/ExportImport/SimpleExportOptions.cs

```diff
@@ -0,0 +1,7 @@
+namespace VirtoCommerce.PricingModule.Core.Model.ExportImport
+{
+    public class SimpleExportOptions
```

Comment:
Could you explain why we need these options in the application config?  Is the platform setting won't enough for this task?

Reply:
The platform has environment managed hidden setting for limitation exporting lines?

Reply:
I will make a task in the platform tech backlog to be able to override runtime platform settings with values are taken from the environment.  I would like to recommend you remove this POCO options and code from the modules.cs that fill the runtime settings from env.